game/Bird.java

```diff
@@ -124,11 +124,11 @@ public void update() {
         if (Resources.IN_TUBE) {
             int leftBird = Resources.BIRD_X_POSITION;
             int rightBird = Resources.BIRD_X_POSITION + Resources.BIRD_WIDTH;
-            int leftTube = (int) Resources.CURRENT_TUBE.getX();
-            int rightTube = (int) Resources.CURRENT_TUBE.getX() + Resources.TUBE_WIDTH;
+            int leftTube = (int) Resources.CURRENT_TUBE.getX()+50;
```

Comment:
I noticed that when I run the game the birds died before reaching the tubes. So I tried to fix the collision detecetion by changing the values.